.changeset/pre.json

```
@@ -0,0 +1,22 @@
+{
+  "mode": "pre",
+  "tag": "premajor",
```

This should be `alpha` or `alpha-v5` I think
```suggestion
  "tag": "premajor",
```

---

done!